### ISSUE TITLE
Support for Laravel 11 breaks support for Laravel 10.

### ISSUE THREAD

**Issue by niladam** [stephenjude/filament-debugger#34]
Hello,

I am wondering why adding support to Laravel 11 had to "drop" laravel 10 from it's support.

Couldn't that have been in the form of `"^10 || ^11" ` allowing both versions as you already did with laravel/telescope previously ?

**Comment by stephenjude** [comment]
No, I can't remember off the top of my head, but I think I added some features that only work in Laravel 11.
